packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -380,6 +421,16 @@ export function ArticleMetadataPanel({
                     setAuthorsFilter('')
                   }}
                   block
+                  renderExtraFooter={() =>
```

Comment:
you seem to repeat yourself here. It might be a good idea to create a new component callled AuthorCheckPicker and move the logic there.